doc/examples.xml

```diff
@@ -309,3 +309,45 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="BananaTreeGraph">
+<ManSection>
+  <Oper Name="BananaTreeGraph" Arg="[filt, ]m, n"/>
+  <Returns>A digraph</Returns>
+  <Description>
+    If <A>n</A> and <A>n</A> are positive integers,
+    then this operation returns the <E>banana tree graph</E> 
+    <M>BananaTreeGraph(<A>m</A>, <A>n</A>)</M>. <P/>
```

Comment:
I think stating it this way is a bit too tautological, I think something like the following would be better 🙂 
```suggestion
    then this operation returns the <E>banana tree graph</E> 
    with parameters <A>m</A> and <A>n</A>, as defined below. <P/>
```